go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -334,8 +352,7 @@ go.app = (function () {
     self.add("state_fever", function (name) {
       return new ChoiceState(name, {
         question: $([
-          "Do you feel very hot or cold? Are you sweating or shivering? When you touch your " +
-          "forehead, does it feel hot?",
+          "Have you been diagnosed with either Obesity, Diabetes, Hypertension or Cardiovascular disease?",
```

Comment:
Why this change? This question now no longer matches with the name of the state, `state_fever`